pom.xml

```diff
@@ -899,7 +900,7 @@
                         <phase>package</phase>
                         <configuration combine.self="append">
                             <sourceDirectory>docs/developer-guide</sourceDirectory>
-                            <outputDirectory>${project.build.directory}/docs/developer-guide/html</outputDirectory>
+                            <outputDirectory>${project.build.directory}/docs/html/developer-guide</outputDirectory>
```

Comment:
Swapped order because the guides link to each other ⇒ it's simpler if they can use trivial relative links than if they're layed out in a more complicated directory hierarchy.